lib/livebook/config.ex

```diff
@@ -106,6 +106,22 @@ defmodule Livebook.Config do
     Application.fetch_env!(:livebook, :shutdown_enabled)
   end
 
+  @doc """
+  Returns the application service name.
+  """
+  @spec app_service_name() :: String.t() | nil
+  def app_service_name() do
+    Application.get_env(:livebook, :app_service_name)
+  end
+
+  @doc """
+  Returns the application service url.
+  """
+  @spec app_service_url() :: String.t() | nil
+  def app_service_url() do
+    Application.get_env(:livebook, :app_service_url)
```

Comment:
```suggestion
    Application.fetch_env!(:livebook, :app_service_url)
```

Then you will need to add a default value of nil here: https://github.com/livebook-dev/livebook/blob/35b3fae3cba4de0414701155e9ac304e1c34b4cf/config/config.exs#L22